### PR TITLE
Content: "Changed Molotovs to need thickener"

### PR DESCRIPTION
**Pull request by faefux**
SUMMARY: Content "Changed Molotovs to need thickener" 

#### Purpose of change
Molotovs generally have thickening agents added to them to make them burn longer/hotter and stick. 

#### Describe the solution
Changed molly recipe to need a thickening agent.

#### Describe alternatives you've considered
Leaving as is or creating two types of molotov. problem with the latter is that there will be no way to differentiate them mechanically, as far as i can tell.

#### Additional context
Feel free to suggest different amounts of the thickening agents if you think they don't seem right.

### CODE REVIEW

**Comment by ZhilkinSerg** [comment]
Summary line should go like this:

```
SUMMARY: Content "Changed Molotovs to need thickener" 
```

File: data/json/recipes/weapon/explosive.json [hunk 1]
```diff
@@ -199,6 +199,7 @@
     "autolearn": true,
     "components": [
       [ [ "rag", 1 ] ],
+      [ ["soap", 5], ["detergent", 5], ["slime_scrap", 5], ["styrofoam_cup", 5] ],
```
Comment (Night-Pryanik) [inline data/json/recipes/weapon/explosive.json]:
```suggestion
      [ [ "soap", 5 ], [ "detergent", 5 ], [ "slime_scrap", 5 ], [ "styrofoam_cup", 5 ] ],
```

**Comment by Orinyau** [comment]
Gasoline+styrofoam (or orange juice concentrate) could be a "Napalm Cocktail"

Motor oil can also be used as a thickening agent, if it is something included in game.

**Comment by Night-Pryanik** [comment]
Why closing?

**Comment by Rivet-the-Zombie** [comment]
Molotovs are supposed to be simple; glass container, fuel, fuse. I'm not a fan of requiring extra materials beyond that, as they don't actually require a thickening agent to work.

**Pull request closed without merging**